evalml/model_understanding/decision_boundary.py

```diff
@@ -0,0 +1,189 @@
+"""Model Understanding for decision boundary on Binary Classification problems."""
+import numpy as np
+import pandas as pd
+
+from evalml.pipelines import BinaryClassificationPipeline
+from evalml.utils import infer_feature_types
+
+
+# these are helper functions to help us calculate the objective values
+def _accuracy(val_list):
+    """Helper function to help us find the accuracy.
+
+    The input expected should be [tp, tn, fp, fn]
```

Comment:
@chukarsten I think we can leave them as private functions for now, since this is currently the only place they are used. In the future, if we want to add them to our objectives, we can.